.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,3 +7,15 @@ resolves #1235
 ### This pullrequest changes
 
 <!-- Please describe what your pullrequest is changing -->
+
+<cut/>
+
+### Pull Request Readiness Checklist
+
+- [ ] I agree to contribute to the project under OpenCV ([BSD](https://github.com/opencv/opencv/blob/master/LICENSE)) license
```

Comment:
It should be changed to:

I agree to contribute the patch under OpenCV (BSD) license: https://github.com/opencv/opencv/blob/master/LICENSE